### PR TITLE
Fix linting and test

### PR DESCRIPTION
**Pull request by amaurymartiny**
This fixes `yarn test` and `yarn lint`

(feel free not to merge if that breaks your base PR)

### CODE REVIEW

File: back/node_watcher/src/tasks/createBlockNumber.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ const createBlockNumber: Task<NomidotBlock> = {
 
     const write = await prisma.createBlockNumber({
       number: blockNumber.toNumber(),
-      authoredBy: authoredBy.toString(),
+      authoredBy: authoredBy.toHex(),
```
Comment (amaurymartiny) [inline back/node_watcher/src/tasks/createBlockNumber.ts]: Still to be discussed, but maybe saving public key is better?

File: back/node_watcher/src/nomidot.spec.ts [hunk 2]
```diff
@@ -16,14 +15,15 @@ describe('Nomidot Tasks', () => {
...
     async (done): Promise<void> => {
       api = await ApiPromise.create({ provider: new WsProvider(parityKusama) });
+
       done();
     }
   );
 
   it('should read a single task', async done => {
-    const intentions = await api.query.staking.validators();
+    const currentValidators = await api.query.session.validators();
```
Comment (amaurymartiny) [inline back/node_watcher/src/nomidot.spec.ts]: staking.validators is a linked map, was taking more than 5 seconds and test failed after that.

File: jest.config.js [hunk 3]
```diff
@@ -2,6 +2,6 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-const config = require('@polkadot/dev/config/jest');
-
-module.exports = Object.assign({}, config);
\ No newline at end of file
+module.exports = {
```
Comment (amaurymartiny) [inline jest.config.js]: The reason tests were failing for me, was that polkadot/dev is using babel, and here we're using tsc. ts-jest is a tsc wrapper around jest

Comment (amaurymartiny) [reply jest.config.js]: We could switch to babel (main argument: faster compile times), but tsc is the official one and imo simpler to use

File: back/node_watcher/src/tasks/createBlockNumber.ts (hunk 1 above)
Comment (amaurymartiny) [reply back/node_watcher/src/tasks/createBlockNumber.ts]: Actually, I'm still hesitant. Addresses will be way easier to inspect

Comment (yjkimjunior) [reply back/node_watcher/src/tasks/createBlockNumber.ts]: maybe we just be redundant and have two columns in there - (public_key, address)?

File: jest.config.js (hunk 3 above)
Comment (yjkimjunior) [reply jest.config.js]: let's stick with tsc

File: back/node_watcher/src/nomidot.spec.ts (hunk 2 above)
Comment (yjkimjunior) [reply back/node_watcher/src/nomidot.spec.ts]: ah, sweet, thank you

**Comment by yjkimjunior** [comment]
thanks a ton!

**Pull request merged**